exercises/routing/README.md

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
At the very beginning of the file we need to provide the overall goal of the exercise and show the gif. Take a look at this [example exercise](https://github.com/rangle/angular-training-slides/tree/master/exercises/structural-directives)